data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1179 @@
+				decline
+
+	npc save
+		on kill
+			set "yvx killed"
+		government "Gegno Vi"
+		personality heroic vindictive staying ramming daring uninterested
+		system "Vorsuke"
+		ship "Rhyolite" "Imtydei"
+	npc save
+		government "Gegno Vi"
+		personality heroic vindictive staying uninterested
+		system "Vorsuke"
+		ship "Schist" "Ess Ouec"
+
+mission "Vi Military Group"
+#	invisible
+	landing
+	to offer
+		has "Passive-Aggressive Observations: done"
+	to complete
+		never
+	on enter "Vorsuke"
+		fail
+
+	npc
+		to spawn
+			has "provoked military"
+		on kill
+			set "yvx killed"
+		government "Gegno Vi (Combative)"
+		personality heroic vindictive ramming daring staying launching
+		system "Vorsuke"
+		ship "Rhyolite" "Imtydei"
+	npc
+		to spawn
+			has "provoked military"
+		government "Gegno Vi (Combative)"
+		personality heroic vindictive staying launching
+		system "Vorsuke"
+		ship "Schist" "Ess Ouec"
+	npc
+		to spawn
+			has "provoked military"
+		government "Gegno Vi (Combative)"
+		personality heroic vindictive staying launching
+		system "Vorsuke"
+		ship "Schist" "Ess Utchec"
+	npc
+		to spawn
+			has "provoked military"
+		government "Gegno Vi (Combative)"
+		personality heroic vindictive staying entering
+		system "Vorsuke"
+		fleet
+			names "gegno vi"
+			variant
+				"Hornfel (Heavy Weapons)" 1
+				"Eclogite" 2
+
+
+mission "Foreign Spectator I"
+#	invisible
+	landing
+	to offer
+#		random < 30
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	to complete
+		never
+	on enter "Eogho"
+		fail
+		"gegno: bgs" ++
+		conversation
+			`Although there is both a Vi and Scin fleet here, neither of them are engaging one another. Instead, they are observing a pair of ships, one from each side, that appear to be challenging one another. Maybe this is some sort of ceremonial event between the Gegno factions. It's probably best not to interfere.`
+
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Vi"
+		personality heroic vindictive ramming staying uninterested
+		system "Eogho"
+		ship "Eclogite" "Tschelgi Dueyu"
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Scin"
+		personality heroic vindictive staying uninterested
+		system "Eogho"
+		ship "Felsic" "Tschelgi Dueich"
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Vi (Neutral)"
+		personality heroic vindictive staying uninterested swarming
+		system "Eogho"
+		fleet
+			names "gegno vi"
+			variant
+				"Hornfel" 2
+				"Eclogite"
+				"Slate"
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Scin (Neutral)"
+		personality heroic vindictive staying uninterested swarming
+		system "Eogho"
+		fleet
+			names "gegno scin"
+			variant
+				"Coesite" 2
+				"Tridymite"
+				"Mica" 2
+
+
+mission "Brief Bystander"
+#	invisible
+	landing
+	destination "Esstch"
```

Comment:
why not just make this mission offer on that planet and fail itself?

Reply:
Could do, yeah.

Reply:
Well, actually, that depends on how we go about the missions offering. Having them offer on each individual planet, with a random chance, is giving the player less of a chance to find the missions than if they were to land multiple times otherwise. This also depends on if these missions should be random at all or not, which I'd like some more input on first.